### PR TITLE
homework Add-even

### PR DESCRIPTION
**Pull request by lubankp**
@lubankp add homework add-even

### CODE REVIEW

**Comment by ziobron** [comment]
🏅 +1 XP for cooperation goes to @lubankp

**Comment by ziobron** [comment]
✅ CI Passed - Task: add-even
🏅 Granted 5 XP for @lubankp

**Comment by ziobron** [comment]
⏱ 3 XP extra for @lubankp for on-time delivery

File: homework/add-even/addEven.hpp [hunk 1]
```diff
@@ -2,7 +2,16 @@
...
-    return -1;
+    
+    auto sum =0;
+
+    for(auto& element : numbers){
+        auto division_rest = element%2;
+        if (division_rest == 0){
+        sum += element;
+        }
+    }
```
Comment (nauka-programowania-MA) [inline homework/add-even/addEven.hpp:13]: Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**